docs/naming-conventions.md

```diff
@@ -0,0 +1,28 @@
+# Naming conventions
+
+The framework provided by the funding extension involves multiple entities, extensions and groups of people that interact which each other. The following list provides a short description of the terms that are mostly used.
+
+**Funding Extension / Funding Framework**: The CiviCRM extension [funding](https://github.com/systopia/funding), as well as the drupal module [civiremote_funding](https://github.com/systopia/drupal-civiremote_funding/), as well as a [set of other extensions and modules](./index.md#we-need-your-support).
+
+**Giving Organization**: The organization that hosts funding programs and allocates money to recipients with eligible applications. The giving organization hosts and administers CiviCRM.
+
+**Funding Portal**: A Drupal website that is created for the interaction with applicants and recipients. The Funding Portal doesn't need to be identical to the Drupal instance that hosts CiviCRM.
+
+-------------
+
+**Applicant / Recipient**: A person or organization that applies for the funding program and possibly receives money. Applicants have access to the Funding Portal. They don't have access to CiviCRM.
```

Comment:
I think we should differentiate the terms `applicant` and `recipient`. In my understanding `recipient` is the organization or person that receives the funding and `applicant` is the one who creates the application.